src/services/cyf-api.js

```diff
@@ -1,27 +1,38 @@
-import mock_students from './mocks/students.json';
+import mock_students from "./mocks/students-1.json";
```

Comment:
It's fine you use a shorter version for your local development, however try not to use `students-1.json` once you publish your code, so in production we'll have the complete dataset

Reply:
I replaced it.